backend/src/main/java/fk/prof/backend/Configuration.java

```diff
@@ -0,0 +1,401 @@
+        @NotNull
+        public final Integer connectionTimeoutMs;
+        @NotNull
+        public final Integer sessionTineoutMs;
+        @NotNull
+        public final Integer maxRetries;
+    }
+
+    public static class BackendAssociationsConfig {
+        @JsonCreator
+        public BackendAssociationsConfig(@JsonProperty("backend.association.path") String associationPath,
+                                         @JsonProperty("load.miss.tolerance") Integer loadMissTolerance) {
+            this.associationPath = associationPath;
+            this.loadMissTolerance = loadMissTolerance;
+        }
+
+        @NotNull
+        public final String associationPath;
+        @NotNull
+        public final Integer loadMissTolerance;
+    }
+
+    public static class DaemonVerticleConfig {
+        @JsonCreator
+        public DaemonVerticleConfig(@JsonProperty("aggregation.window.duration.secs") Integer aggrWindowDurationSecs,
+                                    @JsonProperty("aggregation.window.end.tolerance.secs") Integer aggrWindowEndToleranceSecs,
+                                    @JsonProperty("policy.refresh.offset.secs") Integer policyRefreshOffsetSecs,
+                                    @JsonProperty("scheduling.buffer.secs") Integer schedulingBufferSecs,
+                                    @JsonProperty("work.assignment.max.delay.secs") Integer workAssignmentMaxDelaySecs,
+                                    @JsonProperty("verticle.count") Integer verticleCount) {
+            this.aggrWindowDurationSecs = aggrWindowDurationSecs;
+            this.aggrWindowEndToleranceSecs = aggrWindowEndToleranceSecs;
+            this.policyRefreshOffsetSecs = policyRefreshOffsetSecs;
+            this.schedulingBufferSecs = schedulingBufferSecs;
+            this.workAssignmentMaxDelaySecs = workAssignmentMaxDelaySecs;
+            this.verticleCount = verticleCount;
+        }
+
+        @NotNull
+        public final Integer aggrWindowDurationSecs;
+        @NotNull
+        public final Integer aggrWindowEndToleranceSecs;
+        @NotNull
+        public final Integer policyRefreshOffsetSecs;
+        @NotNull
+        public final Integer schedulingBufferSecs;
+        @NotNull
+        public final Integer workAssignmentMaxDelaySecs;
+        @NotNull
+        public final Integer verticleCount;
+    }
+
+    public static class SerializationWorkerPoolConfig {
+        @JsonCreator
+        public SerializationWorkerPoolConfig(@JsonProperty("size") Integer size,
+                                             @JsonProperty("timeout.secs") Integer timeoutSecs) {
+            this.size = size;
+            this.timeoutSecs = timeoutSecs;
+        }
+
+        @NotNull
+        public final Integer size;
+        @NotNull
+        public final Integer timeoutSecs;
+    }
+
+    public static class StorageConfig {
+        @JsonCreator
+        public StorageConfig(@JsonProperty("s3") S3Config s3Config,
```

Comment:
Is there a pattern being followed for naming as XYZConfig or XYZCfg, if not then we can make them alike.